scripts/duplicate_map.js

```diff
@@ -48,6 +48,26 @@ var ms_tile, lower_range, width, height,
       });
     }
   },
+  create_orig_map = function (repeat, length) {
```

Comment:
If no map file is found(Line 132), it will replace the tiles with tiles from original_map(The map that existed earlier). 
Since we'll usually be adding maps on top, we subtract the difference between the heights of the new map and old map, and copy that part of the old map to the new map. (Line 65)